roles/openshift_node/tasks/ipsec.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Enable ipsec service
+  systemd:
+    name: "ipsec"
+    enabled: yes
+
+- name: add nssdir to ipsec.conf
+  ansible.builtin.lineinfile:
+    path: /etc/ipsec.conf
+    insertafter: 'config setup'
+    line: "\tnssdir=/var/lib/ipsec/nss"
+
+- name: create nssdir
+  file:
+    path: /var/lib/ipsec/nss
+    state: directory
+
+- name: set nss selinux context
+  command: semanage fcontext -a -t ipsec_key_file_t /var/lib/ipsec/nss
```

Comment:
@patrickdillon Do you recall what was happening without this? https://issues.redhat.com/browse/OCPBUGS-30802 says this is failing in an upgrade job from 4.15 to 4.16 on an el8_6 host. Though I can't find any sort of changes to selinux policy that may be related.

Reply:
sorry I missed this comment. this is needed to enable ipsec, but I don't know much more in detail than that. it's failing the upgrade because[ the command is not idempoten](https://bugzilla.redhat.com/show_bug.cgi?id=1609488)t. brent is working on a fix here: https://github.com/openshift/openshift-ansible/pull/12496#pullrequestreview-1968950255